src/ameba/rule/predicate_name.cr

```diff
@@ -40,8 +40,11 @@ module Ameba::Rule
 
     def test(source, node : Crystal::Def)
       if node.name =~ /^(is|has)_(\w+)\?/
+        alternative = $2
+        return unless alternative =~ /^[a-z_][a-zA-Z_0-9]*$/
```

Comment:
I think we need to ignore the "alternative" if it starts with `_`. That would not report the following predicate names:

```crystal
def is__404?
end
```

WDYT?

Reply:
Agreed, suggesting `_404?` over `is__404?` gives no gain. :+1: